app/views/submissions/repo_browser.html.erb

```diff
@@ -62,6 +62,11 @@
         (<span id='current_revision_timestamp_display'>
            <%= l(@revision_timestamp, format: :long_date) %>
         </span>)
+        <br>
+        <%= t('browse_submissions.currently_collected') %> 
+        <%= @grouping.is_collected ? "#" + raw(@grouping.submissions.last.revision_number) + 
```

Comment:
Hmm okay, after looking at this more closely, I have three comments:

1. I'd rather use an `if` here than the ternary operator. 
    ```
      <% if @grouping.is_collected%>
        <% ="#" + @grouping.submissions.last.revision_nubmer %>
      ...
    ```
2. Probably don't need raw for the revision number either
3. I *think* that you can do something like `@grouping.submissions.last.revision_timestamp` to get the timestamp for the submission. I was just looking at that today.